_data/governance.yml

```diff
@@ -0,0 +1,37 @@
+introText: This section is updated every 6 months by the Scala Center.
+cards:
+  - title: "Scala Improvement Process"
+    description: "The SIP is the primary mechanism for evolving the Scala language."
+    expandText: "Process & People"
+    links:
+      - name: SIP Committee Members
+        url: https://docs.scala-lang.org/sips/process-specification.html#the-sip-committee
+      - name: SIP Process Specification
+        url: https://docs.scala-lang.org/sips/process-specification.html
+  - title: "Scala 3 Maintenance"
```

Comment:
I think we should cover both Scala 2 and Scala 3.

Reply:
@lrytz can you help us out here? In terms of the maintenance of Scala 2, what resources can we reference for the current process and people involved?

Reply:
Yeah, those are pretty up to date. If you feel anything is missing, let me know.